File: Lecture004 Solving Pattern Questions (Part-2)/pattern6.py

class solution():
    def pattern(n):
        row=1
        while(row<=n):
            col=row
            while(col>0):
                print(col,end=" ")
                col-=1
            print()
            row+=1

            

if __name__ == '__main__':
    ob=solution
    num = int(input("Enter Number = "))
    ob.pattern(num)

File: Lecture004 Solving Pattern Questions (Part-2)/pattern6.java
import java.util.Scanner;

class pattern6 {
    public static void main(String[] args) {
        int n;

        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        n = sc.nextInt();
        sc.close();

        for (int row = 1; row <= n; row++) {
            for (int col = 1; col <= n; col++) {
                char ch = (char) ('A' + (row - 1));
                System.out.print(ch + " ");
            }

            System.out.println();
        }
    }
}
